.github/workflows/ci.yml

```diff
@@ -29,11 +29,15 @@ jobs:
         if: startsWith(matrix.rust, 'nightly')
         run: cargo check -Z features=dev_dep
       - run: cargo test
-      - name: Force tests with pipe implementation
-        if: startsWith(matrix.os, 'ubuntu')
-        run: cargo test
-        env:
-          RUSTFLAGS: --cfg async_signal_force_pipe_impl
+        if: >
+          !contains(matrix.os, 'windows')
+      # Windows returns 1 regardless of whether or not the signal is intercepted.
```

Comment:
Is this due to the behavior described [here](https://github.com/Detegr/rust-ctrlc/blob/5c3fd397d16dce903bd8a12816c4d121a79e3944/tests/main/harness.rs#L149-L152)?

Reply:
Yes, I believe it is. Though the Windows backend is currently broken for other reasons.